### PR TITLE
Introduce “manage.py initialize” interactive command

### PR DESCRIPTION
**Pull request by theskumar**
- Create/update superuser
- Set wagtail site domains

```
❯ ./manage.py initialize
Provide the details below to initialize Hypha Site. Press enter to keep the default value.

Domain of public site  [hypha.test]:
Domain of apply site  [apply.hypha.test]:
Superadmin Email  [superadmin@hypha.test]:
Superadmin Password  [hypha123]:
Site port [8090]:
>>> Created superuser with email superadmin@hypha.test.
>>> Set public site to hypha.test:8090
>>> Set apply site to apply.hypha.test:8090
```

### CODE REVIEW

File: hypha/core/management/commands/initialize.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import click
```
Comment (frjo) [inline hypha/core/management/commands/initialize.py:1]: This need to be added to requirements-dev.txt

File: hypha/core/management/commands/initialize.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from wagtail.models import Site
+
+
+class Command(BaseCommand):
+    help = "Use."
```
Comment (frjo) [inline hypha/core/management/commands/initialize.py]: A bit short ;-).

**Review by frjo — CHANGES_REQUESTED** [review]
Nice work!

Can we please keep `wagtailsiteupdate.py` but move it to `hypha/core/management/commands/`. I use this myself.

File: hypha/core/management/commands/initialize.py (hunk 1 above)
Comment (theskumar) [reply hypha/core/management/commands/initialize.py:1]: Done

File: hypha/core/management/commands/initialize.py (hunk 2 above)
Comment (theskumar) [reply hypha/core/management/commands/initialize.py]: Done.

File: requirements.txt [hunk 3]
```diff
@@ -55,3 +55,4 @@ tablib==3.2.1
 whitenoise==5.3.0
 xmltodict==0.12.0
 xhtml2pdf==0.2.8
+click==8.1.3
```
Comment (frjo) [inline requirements.txt]: Alphabetical ordering

**Review by frjo — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 3 above)
Comment (theskumar) [reply requirements.txt]: Updated. Along with a few others.

**Review by frjo — APPROVED** [review]

**Pull request merged**